staging/src/k8s.io/component-base/logs/kube-log-runner/kube-log-runner.go

```diff
@@ -26,10 +26,14 @@ import (
 	"os/signal"
 	"strings"
 	"syscall"
+
+	"k8s.io/component-base/logs/logrotation"
 )
 
 var (
 	logFilePath    = flag.String("log-file", "", "If non-empty, save stdout to this file")
+	logFileSize    = flag.Uint("log-file-size", 0, "Useful with log-file, if non-zero, rotate log file when it reaches this size in MB")
+	logFileAge     = flag.Uint("log-file-age", 0, "Useful with log-file-size, if non-zero, remove log files older than this many days")
```

Comment:
Like the above suggestion, can we let users define whether it's a few hours, days or months(e.g. 8h,8d)? Instead of a fixed unit with days as the unit.

Reply:
Same as the above comment, there is a trad off between the flexibility and complexity here. I would prefer to make the parameters to be simple if possible